2101. Detonate the Maximum Bombs/solution.py

```diff
@@ -0,0 +1,13 @@
+class Solution:
+    def maximumDetonation(self, bombs: List[List[int]]) -> int:
+        ### Creating the Bomb Graph ###
+
+        # initialize graph, a graph of bombs
+
+        # iterate through each bomb and check if it can detonate another bomb
+
+            # iterate through each other bomb
+
+                # if the current bomb will detonate another bomb:
+
+                    # create a connection between the two bombs in the graph
```

Comment:
I don't think this is a very efficient algorithm. Is this what everyone else thought of?

Reply:
I think you have to do this. You can halve the run time by looking to see how each pair of bombs affect each other and then not having to check bombs that are lower in index than the current bomb. 

For example:
When you check if bomb 1 will detonate bomb 3, you also check if bomb 3 will detonate bomb 1 since you've already calculated the hypotenuse between their centers. Then when you are processing bomb 3, you only need to check bombs above 3 since you will already have looked at the connection between bomb 3 and bombs 1 and 2.